raiden-dapp/src/services/raiden-service.ts

```diff
@@ -457,6 +462,24 @@ export default class RaidenService {
     });
   }
 
+  private async updatePlannedUserDepositWithdrawals(event: ObservedValueOf<Raiden['events$']>) {
+    if (event.type === 'udc/withdraw/success') {
+      if (event.payload.confirmed === false) {
+        this.store.commit('userDepositContract/clearPlannedWithdrawal');
```

Comment:
Not sure I understand why it has to be cleared for each event if the payload is unconfirmed?

Reply:
What do you mean?
`confirmed === undefined` -> transaction not yet confirmed (not enough blocks on top of it, risk that block with transaction could get reorged)
`confirmed === true` -> transaction confirmed (enough blocks on top, unlikely that blockchain reorgs below)
`confirmed === false` -> block with transaction was reorged an not part of the (longest) blockchain anymore

Still open questions?

Reply:
Ah, that clarifies it, thank you!